auth.py

```diff
@@ -19,7 +20,8 @@ def pwhash(password):
     pwh = 0
     for i, char in enumerate(password):
         pwh += (i + 1) * ord(char)
-    return pwh
+    pws = randint(1,100)+pwh
+    return pws
```

Comment:
it looks like you're attaching the hashed password to the salt. wouldn't it be even more secure to use the salt when generating the hash?